README.md

```diff
@@ -1,3 +1,38 @@
 # Unread Messages Plugin
 
-Adds the ability to count unread messages in ChatEngine.Chats.
+Adds the ability to count unread messages in ChatEngine.Chats
+
+### Quick Start
+
+0. Have ChatEngine instantiated and connected, and have a channel you want to count unread messages on
+```javascript
+const CE = ChatEngineCore.create({
+    publishKey: 'pub-key-here',
+    subscribeKey: 'sub-key-here',
+});
+
+CE.connect('Username');
+CE.on('$.ready', () => { ... }
+```
+
+1. Attach this plugin to the channel you want, in this case global
+```javascript
+CE.global.plugin(ChatEngineCore.plugin['chat-engine-unread-messages']());
+```
+
+2. The plugin needs to be notified when the user is considered active or inactive to properly count messages  
```

Comment:
IMO this plugin is pretty basis because of this. It's really just a counter. Would be cool to add actual functionality to detect if the user has been interacting.